libraries/AP_HAL_ChibiOS/Util.cpp

```diff
@@ -35,6 +35,9 @@
 #if HAL_ENABLE_SAVE_PERSISTENT_PARAMS
 #include <AP_InertialSensor/AP_InertialSensor.h>
 #endif
+#ifndef HAL_BOOTLOADER_BUILD
```

Comment:
Shouldn't this be checking if loving is enabled instead of bootloader?

Reply:
logging enabled is checked via AP_Logging.h, which doesn't build if you include in bootloader build